dashboard/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY setup_nltk.py .
 COPY requirements.txt .
 COPY utility_functions.py .
 RUN pip install -r requirements.txt
-COPY logo_one.png . 
+COPY steampulse_logo.png . 
```

Comment:
If someone other than you redeploys we need to remember to rename the image file